SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -133,9 +133,12 @@ namespace SpatialConstants
 	const Schema_FieldId UNREAL_RPC_PAYLOAD_OFFSET_ID = 1;
 	const Schema_FieldId UNREAL_RPC_PAYLOAD_RPC_INDEX_ID = 2;
 	const Schema_FieldId UNREAL_RPC_PAYLOAD_RPC_PAYLOAD_ID = 3;
+	// UnrealPackedRPCPayload additional Field ID
```

Comment:
I don't think this comment is necessary.

Reply:
There are two schema types, one with (`UnrealPackedRPCPayload`) and one without (`UnrealRPCPayload`) the entity ID. So that nobody expects to see the entity ID in `UnrealRPCPayload`, I decided to be more explicit.